logger_test.go

```diff
@@ -35,6 +36,20 @@ func opts(opts ...Option) []Option {
 	return opts
 }
 
+type stubbedExit struct {
+	Status int
```

Comment:
to distinguish vs being called with `Exit(0)` vs not being called, can we either use a separate bool to track whether it was called, or make `Status` a pointer.